release/lib/skippers.ts

```diff
@@ -1,5 +1,17 @@
 import { Context } from '../@types/custom';
 
+export const skipIfBrowserSame = ({ packages }: Context) => {
+    const pkg = packages && packages.get('extension-browser');
+    const version = pkg && pkg.content.version;
+    const oldVersion = pkg && pkg.oldVersion;
+
+    if (version === oldVersion) {
+        return '';
+    }
+
+    return '';
```

Comment:
you are returning always `return ''` is there any error here?

Reply:
Good catch! The `return` inside the `if` check should have a message stating we're skipping publishing as the browser extension is unchanged.